### PR TITLE
closed last pull request in error; still want to merge these changes!

### PR DESCRIPTION
**Pull request by jperry2-cisco-com**

### CODE REVIEW

File: labs/basic/1.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ For reference treatment of all NB REST APIs to the Cisco APIC-EM controller, see
 [Cisco APIC-EM API Reference Docs](http://devnetapic.cisco.com/)
 
 ## APIC-EM server in this lab
-This lab uses the Cisco APIC-EM controller that the [Cisco DevNet Always-On Sandbox](https://developer.cisco.com/site/devnet/sandbox/) provides. To use a different controller, set the **apicem_ip** variable in the **apicem_config.py** file appropriately. For more information, **see** the "Getting Started" section of this lab.
```
Comment (jperry2-cisco-com) [inline labs/basic/1.md]: this goes to Reservations not to the always on box

File: labs/basic/1.md [hunk 2]
```diff
@@ -175,7 +175,7 @@ For reference treatment of all NB REST APIs to the Cisco APIC-EM controller, see
...
+This lab uses the Cisco APIC-EM controller that the [Cisco DevNet Always-On Sandbox](https://sandboxapic.cisco.com/) provides. To use a different controller, set the **apicem_ip** variable in the **apicem_config.py** file appropriately. For more information, **see** the "Getting Started" section of this lab.
```
Comment (jperry2-cisco-com) [inline labs/basic/1.md]: this is the correct url

File: labs/basic/1.md (hunk 1 above)
Comment (yawming) [inline labs/basic/1.md]: Thanks John this should be [Cisco DevNet Always-On Sandbox](https://developer.cisco.com/sandbox/apic_em)

File: labs/basic/1.md (hunk 2 above)
Comment (jperry2-cisco-com) [inline labs/basic/1.md]: The surrounding text needs to be changed. The sentence immediately following implies that the preceding URL is suitable for use as the apicem_ip variable, but it is not. I think that the simplest change would be to remove the link. That way the text is correct (use the always on box) but it does not point to a URL that cannot be used as the value of apicem_ip var. Thanks for considering this input :-)

**Pull request closed without merging**